git_services/git_services/sidecar/config.py

```diff
@@ -29,8 +26,8 @@ class Config:
     git_proxy_health_port: Union[Text, int] = 8081
 
     def __post_init__(self):
-        self.port = _parse_value_as_numeric(self.port, int)
-        self.git_proxy_health_port = _parse_value_as_numeric(self.git_proxy_health_port, int)
+        self.port = _parse_value_as_int(self.port)
```

Comment:
**question (non-blocking):** Do ports actually have this issue, since they only go to 65535?

Reply:
Probably not but I am not sure at what value k8s decides to convert large ints to scientific notation